### PR TITLE
Crash in EATHook.

### PR DESCRIPTION
**Pull request by zolcsieles**
There are dlls which exports more functions than names.

### CODE REVIEW

File: ExampleProject/ExampleProject/ExampleProject/CMakeLists.txt [hunk 1]
```diff
@@ -22,15 +22,15 @@ if(CMAKE_SIZEOF_VOID_P EQUAL 8)
 	#Link the capstone lib according to config mode
 	target_link_libraries(${PROJECT_NAME} debug ${CAPSTONE_LIB_PATH_64}/Debug/capstone.lib)
 	target_link_libraries(${PROJECT_NAME} optimized ${CAPSTONE_LIB_PATH_64}/Release/capstone.lib)
-	target_link_libraries(${PROJECT_NAME} debug ${PLH_LIB_PATH_64}/Debug/Debug/PolyHook_2.lib)
```
Comment (stevemk14ebr) [inline ExampleProject/ExampleProject/ExampleProject/CMakeLists.txt]: Why was this removed?

File: ExampleProject/ExampleProject/ExampleProject/CMakeLists.txt [hunk 2]
```diff
@@ -22,15 +22,15 @@ if(CMAKE_SIZEOF_VOID_P EQUAL 8)
...
+	target_link_libraries(${PROJECT_NAME} debug ${PLH_LIB_PATH_64}/Debug/PolyHook_2.lib)
 	target_link_libraries(${PROJECT_NAME} optimized ${PLH_LIB_PATH_64}/Release/Release/PolyHook_2.lib)
 elseif(CMAKE_SIZEOF_VOID_P EQUAL 4)
     # 32 bits
 
 	#Link the capstone lib according to config mode
 	target_link_libraries(${PROJECT_NAME} debug ${CAPSTONE_LIB_PATH_86}/Debug/capstone.lib)
 	target_link_libraries(${PROJECT_NAME} optimized ${CAPSTONE_LIB_PATH_86}/Release/capstone.lib)
-	target_link_libraries(${PROJECT_NAME} debug ${PLH_LIB_PATH_86}/Debug/Debug/PolyHook_2.lib)
```
Comment (stevemk14ebr) [inline ExampleProject/ExampleProject/ExampleProject/CMakeLists.txt]: Here too

File: ExampleProject/ExampleProject/ExampleProject/CMakeLists.txt (hunk 1 above)
Comment (zolcsieles) [reply ExampleProject/ExampleProject/ExampleProject/CMakeLists.txt]: It is not related to the EATHook problem (sorry).

The linking of ExampleProject failed because there is no ${PLH_LIB_PATH_...}/Debug/Debug/PolyHook_2.lib, but there is ${PLH_LIB_PATH_...}/Debug/PolyHook_2.lib.
So I removed one "/Debug" from this line.

(I think the Release has the same issue... Release/Release/PolyHook_2.lib)

File: ExampleProject/ExampleProject/ExampleProject/CMakeLists.txt (hunk 2 above)
Comment (zolcsieles) [reply ExampleProject/ExampleProject/ExampleProject/CMakeLists.txt]: The same as at 64 bit version...

**Comment by zolcsieles** [comment]
I want to send the pull request for the "Crash in EATHook." commit.
Then I pushed the "Added build directories to ignore list." commit... which became the part of the pull request... I don't use git too much. :( And sorry for my english.

**Comment by stevemk14ebr** [comment]
These are all fine, i didn't realize it was for the example project 😄 . Build directories needed adding there anyways
